v03_pipeline/lib/vep.py

```diff
@@ -1,13 +1,10 @@
-import hail as hl
+from string import Template
 
-from v03_pipeline.lib.model import DatasetType, Env, ReferenceGenome
+import hail as hl
 
+from v03_pipeline.lib.model import DatasetType, ReferenceGenome
 
-def validate_vep_config_reference_genome(reference_genome) -> None:
```

Comment:
No longer necessary since the reference genome is now embedded as part of the path.